modules/color-picker/main/index.coffee

```diff
@@ -227,7 +227,7 @@ class ColorPicker extends hx.EventEmitter
           if endCallback isnt null then endCallback(elem)
           dragging = false
 
-        dispose: ->
+        dispose = ->
```

Comment:
It seems that disposed is used on 8 places:
```
georginaumov@georgi:~/Documents/hexagonjs$ grep -R disposed --include '*.coffee' *
modules/color-picker/main/index.coffee:        cursorStartPos = elementStartPos = dragging = listening = disposed = null
modules/color-picker/main/index.coffee:          if dragging or not listening or disposed then return
modules/color-picker/main/index.coffee:          if !dragging or disposed then return
modules/color-picker/main/index.coffee:          if not dragging or disposed then return true
modules/color-picker/main/index.coffee:          if disposed then return
modules/color-picker/main/index.coffee:          disposed = true
modules/color-picker/main/index.coffee:          if listening or disposed then return
modules/color-picker/main/index.coffee:          if not listening or disposed then return
georginaumov@georgi:~/Documents/hexagonjs$ grep -R disposed --include '*.coffee' * | wc -l
8
```
but I can remove it. Are you sure? By the way this is my first try with cofeescript so may be I missing something.